app/simulation_api.py

```diff
@@ -8,6 +8,8 @@ class SimulationApi:
 
     def __init__(self, services):
         self.services = services
+        # need the next line to make authentication checks work
+        self.auth_svc = services['auth_svc']
```

Comment:
@wbooth can you confirm/deny if this is needed elsewhere?

Reply:
nope everywhere else is good